### DISCUSSION TITLE
Can't get browser-sync to automatically refresh

### DISCUSSION THREAD

**Discussion by felepp** [tailpress/tailpress#219] [Q&A]
Hello there,

I can't quite get browser-sync to automatically refresh on file changes. I have uncommented the `mix.browserSync` block in webpack.mix.js and inserted my local dev url like this:

```
mix.browserSync({
  proxy: "http://wp-1.test",
  host: "wp-1.test",
  open: "external",
  port: 8000,
});
```

When running `npm run watch` a new browser tab opens just fine and the "browser-sync connected" message appears but I don't get any automatic refresh on file changes. 

Anything I'm missing here?

**Comment by gychem** [comment]
I have the same issue, did you already found a solution?

**Comment by jeffreyvr** [comment]
Try setting `injectChanges` to `true` here:

https://github.com/jeffreyvr/tailpress/blob/master/webpack.mix.js#L26

**Comment by ineptian** [comment]
On windows you need to add the `files` attribute to browserSync optons in `webpack.mix.js`
```html
mix.browserSync({
    proxy: 'whatever.whatever',
    files: ["**/*.php"],
    injectChanges: true
});
```
